3.2/s2i/bin/assemble

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+shopt -s dotglob
+echo "---> Installing application source ..."
+ls -A /tmp/src/* &>/dev/null && mv /tmp/src/* ./
```

Comment:
we always chmod the files to 666 as we layer them into the container, so permissions should not be a problem.